### DISCUSSION TITLE
Add collections icon (box) to icons list

### DISCUSSION THREAD

**Discussion by nelolka** [directus/directus#3636] [Feature Requests]
There is no collections icon in the icons list 🙁
![image](https://user-images.githubusercontent.com/24505418/104108077-72555b00-52ca-11eb-9469-3de347115427.png)

**Comment by benhaynes** [comment]
It's a custom icon we made — we'll need to add it (and other custom icons) to the interface.

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
